src/config/jest.config.js

```diff
@@ -14,17 +14,7 @@ const jestConfig = {
 	resolver: here("jest-resolver.js"),
 	moduleNameMapper: {
 		"\\.(jpg|jpeg|png|gif)$": here("../__mocks__/fileMock.js"),
-		// Needing two versions of date-fns makes things problematic
```

Comment:
Did you really mean to remove that ? Unit tests of orc-shared are failing, you will need to figure that out. Look for Travis build reports.

Reply:
I know that unit tests will fail (in the Date.js).  I am working on an orc-shared that does not fail with that orc-scripts.  There were strange aliases for old date-fns, and we no longer need the version 1.0 with the new date-picker.